utils/utils.go

```diff
@@ -182,3 +185,34 @@ func ReadLocalFile(location string) (string, error) {
 
 	return string(data), nil
 }
+
+// Gets the latest stable release tag from github for a given org name and repo name(in that org)
+func GetLatestReleaseTag(org string, repo string) (string, error) {
+	var url string = "https://github.com/" + org + "/" + repo + "/releases/latest"
+	resp, err := http.Get(url)
+	if err != nil {
+		return "", ErrGettingLatestReleaseTag(err)
+	}
+	defer safeClose(resp.Body)
+
+	if resp.StatusCode != http.StatusOK {
+		return "", ErrGettingLatestReleaseTag(err)
+	}
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return "", ErrGettingLatestReleaseTag(err)
+	}
+	re := regexp.MustCompile("/releases/tag/(.*?)\"")
+	releases := re.FindAllString(string(body), -1)
+	latest := strings.ReplaceAll(releases[0], "/releases/tag/", "")
```

Comment:
Let's not assume that length > 0

Reply:
Length will be always 1, but yeah better to add a if statement.

Reply:
@piyushsingariya what about the case when a repo has no releases?

Reply:
Speaking of releases, the WASM filters need to be built and released - https://github.com/layer5io/wasm-filters/issues/38

Reply:
This is a topic for today's Build and Release meeting.